Pod/Classes/VOKCoreDataManager.m

```diff
@@ -296,6 +327,38 @@ - (NSArray *)importArray:(NSArray *)inputArray forClass:(Class)objectClass withC
     return [returnArray copy];
 }
 
+- (NSArray *)arrayToImportAfterImportingRepeatedUniqueKeys:(NSArray *)repeatedUniqueKeys
+                                   forForeignUniqueKeyPath:(NSString *)foreignUniqueKeyPath
+                                           usingInputArray:(NSArray *)inputArray
+                                                  forClass:(Class)objectClass
+                                               withContext:(NSManagedObjectContext *)contextOrNil
+{
+    NSMutableArray *uniquedInputArray = [NSMutableArray arrayWithArray:inputArray];
+    
+    NSMutableArray *itemsToRemove = [NSMutableArray array];
+    for (id repeatedKey in repeatedUniqueKeys) {
+        for (id item in inputArray) {
```

Comment:
If the goal is to import these in order, should these two `for` loops be flipped?

This and the code above using the counted set make me uncomfortable, but I can't put my finger on why.  Are there tests added to cover all the corners of the intended behavior?